src/lib/Core.jsx

```diff
@@ -317,6 +318,8 @@ const Core = function ({
             {currentQuestionIndex + 1}
             :
           </div>
+
+          <ProgressBar progress={currentQuestionIndex + 1} />
```

Comment:
add a new prop called `showProgressBar` so that people can configure to show it or not. By default, please set it as true.